server/templates/place.html

```diff
@@ -25,6 +25,11 @@
 {% block head %}
 <link rel="stylesheet" href={{url_for('static', filename='css/place_page.min.css' , t=config['VERSION'])}}>
 <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" />
+{% if category == '' %}
+<link rel="canonical" href="https://datacommons.org/place/{{ place_dcid }}" />
```

Comment:
Should we use the site's actual hostname here (in case this is a custom DC instance)

Reply:
Great callout. Switched to using actual hostname.